exercises/bob/test/Test.hx

```diff
@@ -0,0 +1,91 @@
+package;
+
+using buddy.Should;
+
+class Test extends buddy.SingleSuite {
+    public function new() {
+        describe("Bob", {
+            final SURE     = "Sure.";
+            final CHILL    = "Whoa, chill out!";
+            final CALM     = "Calm down, I know what I'm doing!"; 
+            final FINE     = "Fine. Be that way!";
+            final WHATEVER = "Whatever.";
+
+            it("stating something", {
+                Bob.hey("Tom-ay-to, tom-aaaah-to.").should.be(WHATEVER);
+            });
+            it("shouting", {
```

Comment:
In general, tracks that support it have all tests ignored except for the first test. See for example the [C# bob tests](https://github.com/exercism/csharp/blob/master/exercises/bob/BobTests.cs#L13). The idea is that it forces students to solve tests one at a time, effectively engaging in TDD. Is this something that is supported by the test framework?

Reply:
I can add a pending call which will skip over a test.  Then users can just comment it out to run the test. 
```haxe
            it("stating something", {
                Bob.hey("Tom-ay-to, tom-aaaah-to.").should.be(WHATEVER);
            });
            it("shouting", {
                pending("Skipping");
                Bob.hey("WATCH OUT!").should.be(CHILL);
            });
```
```
XPPPPPPPPPPPPPPPPPPPPPPPP
Bob
  stating something (FAILED)
    Not Implemented
      @ /home/davidbruce/Git/haxe/buddy/src/buddy/SuitesRunner.hx:519
      @ test/Test.hx:16
      @ src/Bob.hx:5
  shouting (Pending)
    test/Test.hx:19: Skipping
  shouting gibberish (Pending)
    test/Test.hx:23: Skipping
  asking a question (Pending)
    test/Test.hx:27: Skipping
  asking a numeric question (Pending)
```

Reply:
Yes that would be great!